tests/test_unclassified_job.py

```diff
@@ -34,6 +36,8 @@ def test_open_unclassified_failure_log(self, mozwebqa):
         # Open the job log and verify there is content
         resultset_page = ResultsetPage(mozwebqa)
         resultset_page.go_to_page()
+        Assert.not_equal("0","resultset_page.return_unclassified_failure_count")
```

Comment:
In fact, I think this should also check that the count is greater than 1 (or greater than zero).